src/app/feature/consultant/components/consultant-picture-edit/consultant-picture-edit.component.ts

```diff
@@ -48,9 +50,15 @@ export class ConsultantPictureEditComponent implements OnDestroy {
         urlProfileImage: this.croppedImage
       };
 
-      this.consultantStore.updateConsultant(updatedData)
-        .pipe(takeUntil(this.destroy$))
-        .subscribe(() => this.close());
+      if(this.croppedImage){
+        this.consultantStore.updateConsultant(updatedData)
+           .pipe(takeUntil(this.destroy$))
+           .subscribe(() => this.close());
+           this.notification.openUpdatedPictureSnackBar();
```

Comment:
Please put this inside of the subscription. If the updateConsultant fails then the success snackbar will still show because there's no error catching on the observable.